.cirrus/tasks.yml

```diff
@@ -156,14 +153,12 @@ chart_static_compatibility_test_task:
     memory: 1Gb
   <<: *CLONE_SCRIPT_TEMPLATE
   script:
-    - helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
-    - helm repo add bitnami-pre2022 https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
+    - cd "${CIRRUS_WORKING_DIR}" && ./.cirrus/build_chart_dependencies.sh charts/sonarqube
```

Comment:
cd "${CIRRUS_WORKING_DIR} this is not needed as the script start in CIRRUS_WORKING_DIR